StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/ConfirmButton.swift

```diff
@@ -63,9 +63,6 @@ class ConfirmButton: UIView {
         get {
             return buyButton.succeededBackgroundColor
         }
-        set {
-            buyButton.succeededBackgroundColor = newValue
-        }
```

Comment:
Dead code, was never hit.

Reply:
Can we delete the entire `var succeededBackgroundColor: UIColor`?

Reply:
Hmm yep can remove that too.